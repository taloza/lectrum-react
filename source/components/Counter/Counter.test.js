import React from 'react';
import Counter from './';
import dom from 'react-test-renderer';

const renderTree = dom.create(<Counter count = { 5 }/>).toJSON();

test('Counter Component should  to its snapshot', () =>{
    expect(renderTree).toMatchSnapshot();
});
