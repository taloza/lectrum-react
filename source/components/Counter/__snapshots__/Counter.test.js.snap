// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter Component should  to its snapshot 1`] = `
<section
  className="counter"
>
   Posts count: 
  5
</section>
`;
